### ISSUE TITLE
DNN-10818 - value cannot be null when using facebook auth

### ISSUE THREAD

**Issue by vincentparrett** [dnnsoftware/Dnn.Platform#2284]
## Description of bug
Error when logging in using facebook provider, login does not happen.

## Steps to reproduce
Configure facebook auth provider, attempt to login using facebook. 

## Current result
Login does not occur, after allowing the app to use facebook, I am redirected to 
```
domain.com/Default.aspx?tabid=286&error=An%20unexpected%20error%20has%20occurred&content=0#=
```
## Expected result
User should be logged in.

## Error log
```
AbsoluteURL:/Default.aspx
DefaultDataProvider:DotNetNuke.Data.SqlDataProvider, DotNetNuke
ExceptionGUID:b7cf5172-4daf-4480-b5a1-cf9bff78257c
AssemblyVersion:9.2.2
PortalId:0
UserId:-1
TabId:211
RawUrl:/log-in?code=AQAitn8jU6vcOo3xIwuFaTzJSAKAROaKOveCcmP2iAT0prKhyjWbc-NdgJpwS949wfToaCcE4Agg2d2bFPk8jc_a9DkopsjLBsLoyFVdXRqWDr8S-cVGbRrL5CoU_uoMvGNXxJ4nHCD24_21KV7Qk8zl61lirgrO1_LgrDApWtq6SEVrIFRaqIb9XdJBLDF7-0sJtXbmRcnIriV9zr4T2_ux-zGy_7OpNwHZGtSE-hF_wOcmA_E-iiA
Referrer:https://www.domain.com/log-in?returnurl=%2f
UserAgent:Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36
ExceptionHash:Jdlvr4Wmmn5mMSCri9024eZD6Y4=
Message:Value cannot be null. Parameter name: value
StackTrace:
InnerMessage:Value cannot be null. Parameter name: value
InnerStackTrace:
at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings) at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings) at DotNetNuke.Services.Authentication.OAuth.OAuthClientBase.GetCurrentUser[TUserData]() at DotNetNuke.Services.Authentication.OAuth.OAuthLoginBase.OnLoad(EventArgs e) at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Control.LoadRecursive() at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
Source:
FileName:
FileLineNumber:0
FileColumnNumber:0
Method:
Server Name: WEB01
```
## Affected version

* [x] 9.2.2
Using 9.2.2 RC (to unblock registration issue fixed in 9.2.2). 

## Affected browser
* [x] Chrome
* [x] Firefox
Others not tested.

**Comment by thabaum** [comment]
https://github.com/dnnsoftware/Dnn.Platform/issues/2257

**Comment by thabaum** [comment]
Maybe keep these in one conversation I have  a few up... this way we can close all these tickets at once I believe they all related to OAuthV3 support as it is set up for OAuthV2.  

Need to study the changes or what might have broke along the way from DNN 8 to DNN9.

**Comment by valadas** [comment]
I am closing this issue as a duplicate of #2257 please follow #2257 so we have only one issue to track for this problem
